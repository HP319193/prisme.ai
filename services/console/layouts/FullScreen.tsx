import { FC } from 'react';

export const FullScreen: FC = ({ children }) => {
  return (
    <div className="flex justify-center align--center min-h-screen">
      {children}
    </div>
  );
};

export default FullScreen;
