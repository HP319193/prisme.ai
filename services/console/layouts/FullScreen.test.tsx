import FullScreen from './FullScreen';
import renderer from 'react-test-renderer';

it('should render', () => {
  const root = renderer.create(<FullScreen />);
  expect(root.toJSON()).toMatchSnapshot();
});
