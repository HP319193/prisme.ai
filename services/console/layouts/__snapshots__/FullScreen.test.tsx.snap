// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="flex justify-center align-center min-h-screen"
/>
`;
